src/findbugs/excludesFilter.xml

```diff
@@ -0,0 +1,76 @@
+<FindBugsFilter>
```

Comment:
I updated parent POM for flexibility which also ended up including some other changes, such as the maven `findbugs` plugin being introduced. I fixed half of the reported issues and used this filter to ignore unimportant ones.